packages/calcite-components/src/components/block/resources.ts

```diff
@@ -43,3 +43,5 @@ export const ICONS = {
   valid: "check-circle",
   invalid: "exclamation-mark-triangle",
 } as const;
+
+export const defaultMenuPlacement = "bottom-end";
```

Comment:
There’s a [`defaultMenuPlacement` in `floating-ui.ts`](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/utils/floating-ui.ts#L273) as well. How should we decide when to use it vs creating a separate one?

Reply:
They are both positioned bottom but for dropdown/combobox bottom-start makes more sense whereas for action-menu's mostly positioned at the end bottom-end makes more sense. 

I can put this const in floating-ui.ts